### PR TITLE
Update teams-powershell-release-notes.md

### PR DESCRIPTION
**Pull request by rajgodwin**
2.5.2 release updates

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
 
 | Date | Version | Updates |
 |------- | -------------------- | ------------------------------ |
+| September 2021 | [2.5.2](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.2) |<li>Note: Starting from this version, the release notes will also be published on PowerShell Gallery along with the module itself - to reduce the delay in availability of release notes.</li><li>Releases [Get|Set|Grant|New|Remove]-CsTeamsEnhancedEncryptionPolicy cmdlets.</li><li>Removes [Get|Set|New|Remove]-CsTenantBlockedNumberExceptionPattern cmdlets.</li><li>Fixes the bug where MicrosoftTeams module lookup failed if made a nested module of another custom PowerShell module. Now, MicrosoftTeams cmdlets are available even if its a nested module of another module.</li><li>Releases [Get|New|Complete|Clear]-CsOnlineTelephoneNumberOrder cmdlets.</li><li>Releases Get-CsOnlineTelephoneNumberCountry, Get-CsOnlineTelephoneNumberType cmdlets.</li><li>Fixes the failure that occured when retrying login with Connect-MicrosoftTeams after an incorrect sign-in attempt.</li>|<li>Fixes Add-TeamChannelUser and Remove-TeamChannelUser failing for private channel.</li>
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| September 2021 | [2.5.2](https://www.powershellgallery.com/packages/MicrosoftTeams/2.5.2) |<li>Note: Starting from this version, the release notes will also be published on PowerShell Gallery along with the module itself to reduce the delay in availability of release notes.</li><li>Releases (Get, Set, Grant, New, Remove) -CsTeamsEnhancedEncryptionPolicy cmdlets.</li><li>Removes (Get, Set, New, Remove) -CsTenantBlockedNumberExceptionPattern cmdlets.</li><li>Fixes the bug where the Microsoft Teams module lookup failed if it was made a nested module of another custom PowerShell module. Now, Microsoft Teams cmdlets are available even if its a nested module of another module.</li><li>Releases (Get, New, Complete, Clear) -CsOnlineTelephoneNumberOrder cmdlets.</li><li>Releases Get-CsOnlineTelephoneNumberCountry and Get-CsOnlineTelephoneNumberType cmdlets.</li><li>Fixes the failure that occured when retrying sign-in with Connect-MicrosoftTeams after an incorrect sign-in attempt.</li>|<li>Fixes Add-TeamChannelUser and Remove-TeamChannelUser failing for private channel.</li>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rajgodwin, kindly commit the suggested changes as indicated (escape character before "|" is essential). Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@brandber PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@rajgodwin - Can you please check this PR and merge it?

**Pull request merged**